angular-workspace/projects/ni/nimble-angular/rich-text/editor/nimble-rich-text-editor.directive.ts

```diff
@@ -1,8 +1,10 @@
 import { Directive, ElementRef, EventEmitter, HostListener, Input, Output, Renderer2 } from '@angular/core';
```

Comment:
There are a large number of changes in #1683 that have not been merged here yet. I think I need to wait for that merge for review.